diffsync/enum.py

```diff
@@ -73,3 +73,12 @@ class DiffSyncStatus(enum.Enum):
     SUCCESS = "success"
     FAILURE = "failure"
     ERROR = "error"
+
+
+class DiffSyncActions:  # pylint: disable=too-few-public-methods
```

Comment:
I don't like having a "not a real Enum" class in `enum.py`. Can you provide more context as to why we're not using an actual `enum.Enum` class here, unlike all the other classes in this file?

Reply:
I guess it was my inexperience with the enum, somehow I expected them to behave like the `ChoiceSet` in Nautobot and to translate directly to the real value
```
>>> DiffSyncActions.CREATE
"create"
``` 

I guess the right solution is just to use `DiffSyncActions.CREATE.value` instead but honestly it's not 100% clear to me what is the value to use the real enum class instead of a simple dataclass.